library/Icinga/Application/Modules/Module.php

```diff
@@ -371,6 +381,37 @@ protected function dashboard($name, array $properties = array())
         return $this->paneItems[$name];
     }
 
+    /**
+     * Get this module's provided dashlets
+     *
+     * @return DashletContainer[]
+     */
+    public function getDashlets()
+    {
+        $this->launchConfigScript();
+        return $this->dashletItems;
+    }
+
+    /**
+     * Add or get a dashlet
+     *
+     * @param string $name
+     * @param Url|string $url
+     * @param array $properties
+     *
+     * @return DashletContainer
+     */
+    protected function provideDashlet($name, $url, array $properties = [])
```

Comment:
Please add a type declaration to param `$name`

Reply:
Should I have to add param type declaration of `string, int, bool` in other code places too? I was thinking we do this only `icingadb-web`!

Reply:
icingadb-web already ditches support for php 5.6 7.0 and 7.1. Web 2 will as well with the update to 2.11.